src/component/TooltipBoundingBox.tsx

```diff
@@ -138,18 +112,21 @@ export class TooltipBoundingBox extends PureComponent<TooltipBoundingBoxProps, S
     };
 
     return (
-      // This element allow listening to the `Escape` key.
-      // See https://github.com/recharts/recharts/pull/2925
-      <div
-        tabIndex={-1}
-        className={cssClasses}
-        style={outerStyle}
-        ref={node => {
-          this.wrapperNode = node;
-        }}
-      >
-        {children}
-      </div>
+      <g>
+        <foreignObject x="0" y="0" width="100%" height="100%" style={{ pointerEvents: 'none' }}>
```

Comment:
Any downsides of doing this? Doesn't the X and Y matter?

Reply:
No downside as far as I can tell.